scripts/package/package-debian.sh

```diff
@@ -97,28 +97,43 @@ create_debian_package(){
 test_debian_package(){
     header "Testing debian package"
     
-    # Set LAST_VERSION_URL to enable upgrade tests
-    export LAST_VERSION_URL="https://dotnetcli.blob.core.windows.net/dotnet/$CHANNEL/Installers/Latest/dotnet-ubuntu-x64.latest.deb"
+    install_bats
```

Comment:
That's a fair question and I hadn't thought of it in this frame.

I'm not really in favor of bats, it doesn't give us much and is only acquirable via git. 
Bringing dotnet test in the picture has more complexity but I think having it as the test runner, with a graceful fallback would be better than bats. 

What I want to avoid is package_tool having a dependency inside the repo itself (so it can remain conceptually as an independent tool). So far that independence hasn't slowed us down and I think it has value over the long term.

If it had a dependency on the dotnet test binary (via apt-get or whatever kind of install) I think that makes sense. The test would be more useful anyways since it will integrate better with build systems.

So an invocation of package_tool could be `package_tool -t` where -t specifies to compile and drop a test assembly next to the deb package. What I'm not sure of is whether the package_tool should auto run this assembly (via `dotnet test`) or if that should be an external action.

What do you think?